index.js

```diff
@@ -57,6 +73,34 @@ router
     ctx.status = 202
     ctx.body = 'OK'
   })
+  .get('/cloudflareError.png', async ctx => {
+    if (ctx.query['cloudflareErrorType'] == undefined || ctx.query['rayID'] == undefined || ctx.query['clientIP'] == undefined) {
+      ctx.status = 404
+    }
+    else if (!['500', '1000'].includes(ctx.query['cloudflareErrorType'])) {
+      console.error('Unregistered error type')
```

Comment:
Best to always include the actual given value, if you want to be able to use these logs as a means to debug in the future. (Same applies to the logging below.)